CHANGELOG.md

```diff
@@ -5,8 +5,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [0.8.3] - 2023-08-24
 ### Added
 - Added support for distributing debian packages.
+- The `azureauth ado pat` subcommand now supports additional Azure DevOps scopes.
```

Comment:
It totally makes sense we update the CHANGELOG, but should we create a separate PR for the version number?

Reply:
We can. Or we can immediately generate the release right after this merges. I'm happy either way. 😄